### PR TITLE
sorting and updated view

### PR DESCRIPTION
**Pull request by adampaulberg**
- no ring should not display launch key selector
- development ring should display launch key selector
- deployment rings other than development should not show the launch key selector, but should select all the values behind the scenes for it's tier and above

- on the job page there should be a new section under the first tab of the 3-install-many tab that shows only the newly selected launch keys, in alphabetical order.

### CODE REVIEW

File: src/server/mappers/env.sql [hunk 1]
```diff
@@ -1,19 +1,22 @@
 -- insert
 insert into env (
   envId, envName, envDesc, status, remoteType, remoteId, config,
-  deployedBy, deployedOn, deployedUntil, deployedNotes, deployedBranch, deployedJobId, host, updatePath
+  deployedBy, deployedOn, deployedUntil, deployedNotes, deployedBranch, 
+  deployedJobId, host, updatePath
 )
```
Comment (jakebrown58) [inline src/server/mappers/env.sql]: the changes to this file look to be purely cosmetic, right?

File: src/server/controllers/envs.js [hunk 2]
```diff
@@ -98,18 +98,19 @@ exports.pause = function pause(req, res, next) {
...
-    });
+  
+  try {
+    res.result = await redeployService.redeploy(data, opts);
+  } catch(err) {
+    res.err = err;
+  }
+  
+  return next();
 };
```
Comment (jakebrown58) [inline src/server/controllers/envs.js]: nice

File: src/server/models/env.js [hunk 3]
```diff
@@ -24,7 +24,8 @@ schema = {
     { "name": "deployedJobId" },
     { "name": "runstate" },
     { "name": "machines" },
-    { "name": "updatePath" }
+    { "name": "updatePath" },
+    { "name": "newLaunchKeys" }
   ]
```
Comment (jakebrown58) [inline src/server/models/env.js]: why do we need a new field here?  or was this missing from before?   it seems like from a process perspective, we aren't making a change in this story - we're just auto-selecting all the unchecked boxes for you and hiding the panel.

the data for the launch keys is already stored *in the config?* at that point as 
"LAUNCHKEY|MyCoolFeature=ThisFeatureSolvesAllTheProblemsOfTheWorld"

File: src/server/mappers/env.sql (hunk 1 above)
Comment (adampaulberg) [inline src/server/mappers/env.sql]: yeah, matching up line length with the previous line

**Pull request merged**